pkg/cluster/cluster.go

```diff
@@ -206,3 +212,73 @@ func GetClusterAddOns(connection *sdk.Connection, clusterID string) ([]*AddOnIte
 
 	return clusterAddOns, nil
 }
+
+// DoSurvey will ask user to choose one if there are more than one clusters match the query
+func DoSurvey(clusters []*clustersmgmtv1.Cluster) (cluster *clustersmgmtv1.Cluster, err error) {
```

Comment:
`DoSurvey` function is more related to cli options and not to cluster related functions that exists in this package.
Please consider using `GetCluster` which will give you a more fine grained result and you won't need to use `DoSurvey` function as part of the new command.
Also, `survey.v1` is outdated and we moved to use `v2` in other commands.

Reply:
These functions were moved verbatim from the existing ocm cluster login, so that the two commands share these code paths.

Reply:
Please see https://github.com/openshift-online/ocm-cli/pull/119/commits/60af13a2f2a99481dfb2f18bacd8b41886ddccb4 where these were just moved out from `ocm cluster login` so they could be reused in `ocm cluster tunnel`.

Reply:
Can you use the `FindCluster` function instead?

Reply:
https://github.com/openshift-online/ocm-cli/blob/master/pkg/cluster/cluster.go#L44